src/vsr/replica_test.zig

```diff
@@ -550,6 +550,8 @@ test "Cluster: repair: crash, corrupt committed pipeline op, repair it, view-cha
     t.run();
 
     a0.stop();
+    t.run();
```

Comment:
Not sure what's the appropriate fix here tbh. Perhaps its `.stop()` which should be delivering/dropping any residual messages?

Reply:
I don't understand what you're referencing. And what is this stop for?

Reply:
Sorry, I think I forgot to type the beginning of the comment here :)


So without this .run(), the test now fails. b1 *somehow* gets prepare for 30, so the cluster downs actually get stuck.

what I think is happening here is that, while we shut down a, we still get a repairing prepare from a in the packet simulator. So, when `pass_all` below, this prepare gets delivered, despite a being down (didn’t verify that this is exactly what’s happening).

🤔 hm, I guess I still don’t understand why the `<` change has this exact effect here, will revisit on Monday

Reply:
Took a closer look here. It indeed is the case that, *after* we call `a0.stop()`, there could still be a pending `.prepare` message in-flight, and that a subsequent `t.run()` gets rid of it. 

Now, why _exactly_ the `<=` change causes the `.prepare` to be queued, is unclear to me. But it seems that it's basically a "timing" issue. 

`<=` doesn't cause any meaningful changes. In particular, the handling of `.recovering_head` for `op=30` is the same. What is different is the handling of an earlier `.start_view` for `op=0`. We either ignore this start view (with `<=`), or trigger a repair (with `<`). The repair is no-op, _except_ that it resets repair timer. And it seems that the timer ultimately affects whether a repairing prepare for `op=30` gets into the window around the crash of the primary. 

So, yeah, I think our testing infra (or maybe just a specific test) is buggy, in a sense that, when you do `a.stop()`, you can't be sure which messages would be delivered.

Reply:
went for a slightly cleaner tacktical fix of cutting the network in addition to stoping the replica itself.